src/map/chrif.c

```diff
@@ -324,6 +324,9 @@ bool chrif_save(struct map_session_data *sd, int flag) {
 	if( sd->save_quest )
 		intif->quest_save(sd);
 
+	if (VECTOR_LENGTH(sd->storage))
```

Comment:
I don't understand what you mean here, if you could explain a little more in detail please.